.travis.yml

```diff
@@ -19,11 +19,6 @@ matrix:
 #      env:
 #        - PYTHON_VERSION=3.4.5
```

Comment:
I was just working on some scripts for the 1.0 release... I'll see if I can run them on the release branch too like the recent x86_64 0.10.0.post3 wheel.